code/modules/hydroponics/hydroitemdefines.dm

```diff
@@ -211,7 +212,7 @@
  * Returns the formatted output as text.
  */
 /obj/item/plant_analyzer/proc/scan_plant_chems(obj/item/scanned_object)
-	var/returned_message = "[span_info("*---------*\nThis is \a <span class='name'>[scanned_object]")].\n"
+	var/returned_message = "*---------*\nThis is a [span_name("[scanned_object.name]")].\n"
```

Comment:
Does this work gramatically when you are scanning apples?

> This is a apple.

Reply:
You're correct about that, for some reason I noticed it in testing but thought `a apple` was fine.